src/components/Partners/Partners.tsx

```diff
@@ -73,6 +73,25 @@ Plusieurs acteurs soutiennent France Chaleur Urbaine : ils contribuent au dével
           <Icon name="ri-arrow-right-circle-line" size="xl" />
         </Arrow>
       </Wrapper>
+      <ButtonGroup isInlineFrom="xs" align="center">
+        <Button
+          onClick={() =>
+            window.open(
+              'mailto:france-chaleur-urbaine@developpement-durable.gouv.fr'
+            )
+          }
+        >
+          Rejoindre notre réseau
+        </Button>
+        <Button
+          secondary
+          onClick={() =>
+            window.open('/documentation/FCU_dossier_presentation.pdf')
+          }
+        >
+          Notre dossier de présentation
+        </Button>
```

Comment:
Is it possible, with the React-DSFR to display a link (<a>) like a Button ? (>> For/because SEO)